perf/benchmark/configs/istio/mixer/installation.yaml

```diff
@@ -4,6 +4,8 @@ spec:
   components:
     telemetry:
       enabled: true
+    citadel:
+      enabled: true
```

Comment:
You shouldn't need citadel - it does not even exist in master. Or is this meant for 1.5?

Reply:
yep I know it is removed from master and it is for 1.5. The reason why I do not just directly merge to 1.5 branch is because sometimes user would just use the scripts from master of tools/ directly but actually want to test istio 1.5, but I think it is a noop for master though so I created the PR directly to master